src/rl.py

```diff
@@ -767,10 +768,18 @@ def evaluate_proof(args: argparse.Namespace,
             coq_serapy.summarizeContext(coq.proof_context)
         eprint(f"Trying predictions {[action.prediction for action in actions]}",
                guard=args.verbose >= 2)
-        action_scores = evaluate_actions(coq, v_network, path,
+        if args.evaluate_baseline :
+            sorted_action = sorted(actions, key = lambda x : x.certainty)
+            best_action,best_score = None, float("-Inf")
+            for action in sorted_action :
+                if action_result(coq, path, action.prediction, args.verbose) :
+                    best_action, best_score = action, action.certainty 
+                    break
+        else :
+            action_scores = evaluate_actions(coq, v_network, path,
                                          [action.prediction for action in actions],
```

Comment:
These two lines should be indented to match the beginning of the first argument in the line above.